frontend/src/actions/index.ts

```diff
@@ -0,0 +1 @@
+export { getEmbeddingPoints, getEmbeddingPointDetails } from "./api/embedding";
```

Comment:
not sure this is needed, can't you just get the functions for the embeddings.ts  file?